### PR TITLE
readme.md has been updated

### PR DESCRIPTION
**Pull request by sf-jeff-lim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,93 @@
...
+
+This section shows the prerequisites you need to check to use Sendbird SyncManager for Android.
+
+### Requirements 
+
+The minimum requirements for SyncManager for Android are:
+
+- `Android 4.0 (API level 14) or higher`
+- `Java 7 or higher`
+- `Gradle 3.4.0 or higher`
```
Comment (hoon-sung) [inline README.md]: I think it'd good to match the requirements from the docs, as below
```
Android 4.0 (API level 14) or higher
Java 8 or higher
Sendbird Chat SDK for Android 3.0.149 or higher (3.0.149 is embedded in SyncManager 1.1.27. If you want to use a higher version, you can directly declare a dependency on the higher version.)
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,93 @@
...
+
+```gradle
 repositories {
     maven { url "https://raw.githubusercontent.com/sendbird/sendbird-syncmanager-android/master/" }
 }
 dependencies {
-    // SyncManager
-    implementation 'com.sendbird.sdk:sendbird-syncmanager:1.1.28'
+    // SyncManager SDK for Android (Latest, embeds sendbird sdk 3.0.147)
+    implementation 'com.sendbird.sdk:sendbird-syncmanager:1.1.26'
```
Comment (hoon-sung) [inline README.md]: The latest version is 1.1.28, and it embeds sendbird sdk 3.0.150)

File: README.md [hunk 3]
```diff
@@ -1,33 +1,93 @@
...
+
+    // Chat SDK for Android (If you want to use higher version than the version embedded in the sync manager)
+    implementation 'com.sendbird.sdk:sendbird-android-sdk:3.0.149'
```
Comment (hoon-sung) [inline README.md]: Latest version is 3.0.150

File: README.md (hunk 1 above)
Comment (hoon-sung) [reply README.md]: Actually, the doc should be updated as the last line should be 
"Sendbird Chat SDK for Android `3.0.150` or higher (`3.0.150` is embedded in SyncManager `1.1.28`. If you want to use a higher version, you can directly declare a dependency on the higher version.)"

Will create PR for updating the doc as well.

Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (sf-jeff-lim) [reply README.md]: Done!

**Review by hoon-sung — APPROVED** [review]

**Pull request merged**